project/project/PlayCrossCompilation.scala

```diff
@@ -1,24 +1,22 @@
 import sbt._
 import Keys._
 import uk.gov.hmrc.playcrosscompilation.AbstractPlayCrossCompilation
-import uk.gov.hmrc.playcrosscompilation.PlayVersion.{Play25, Play26}
+import uk.gov.hmrc.playcrosscompilation.PlayVersion.Play26
 
-object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play25) {
+object PlayCrossCompilation extends AbstractPlayCrossCompilation(defaultPlayVersion = Play26) {
 
   lazy val version: String = playVersion match {
-    case Play25 => "2.5.19"
```

Comment:
Did v2.0.0 of the cross-compile plugin kill 2.5?

Reply:
Yep